tests/ui/macros/macro-path-prelude-fail-4.stderr

```diff
@@ -3,6 +3,14 @@ error: expected derive macro, found built-in attribute `inline`
    |
 LL | #[derive(inline)]
    |          ^^^^^^ not a derive macro
+   |
+help: Remove from the surrounding `derive()`
```

Comment:
Could you add a new test case that has some of the trickier cases for this change? e.g. two derives, macro (to confirm it doesn't trigger), etc.

Reply:
As far as I can tell, this isn't valid. Was there something else you meant with macro?
```rust
macro_rules! foo {
    () => {
        Debug
    };
}

#[derive(foo!())]
struct V;
```

Reply:
```rust
macro_rules! foo {
    ($foo:expr) => {
		#[derive($foo)]
		struct V;
    };
}
```

Something more like this?